context/app/static/js/components/Search/__tests__/SearchWrapper.spec.js

```diff
@@ -0,0 +1,34 @@
+import { getByPath, getOrder } from '../SearchWrapper';
+
```

Comment:
Testing utility functions is great. We should also test the `SearchWrapper` component as well in a later PR.

Reply:
Let me file an issue for that... Getting into mocking the api responses was more than I wanted to tackle right now. https://github.com/hubmapconsortium/portal-ui/issues/1038